File: VB/DrillDownSample/MainViewModel.vb
Public Class MainViewModel
    Dim _branches As IReadOnlyList(Of DevAVBranch)
    Public ReadOnly Property Branches As IReadOnlyList(Of DevAVBranch)
        Get
            Return _branches
        End Get
    End Property

    Public Sub New()
        _branches = New BranchDAO().GetBranches()
    End Sub

End Class

File: CS/DrillDownSample/MainViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Runtime.CompilerServices;
using System.Text;
using System.Threading.Tasks;

namespace DrillDownSample {
    class MainViewModel {
        public IReadOnlyList<DevAVBranch> Branches { get; } 

        public MainViewModel() {
            Branches = new BranchDAO().GetBranches();
        }
    }
}
